docs/source/api-advanced.rst

```diff
@@ -8,21 +8,22 @@ API GoPiGo3 - Advanced
 Requirements
 ============
 
-Before using this chapter's classes, you need to be able to import the following modules.
+Before using this chapter's classes, you need to be able to import the following module.
 
 .. code-block:: python
 
    import easygopigo3
-   import gopigo3
 
-If you have issues importing these 2 modules, then make sure that:
+If you have issues importing this module, then make sure that:
 
    * You've followed the steps found in :ref:`Getting Started <getting-started-chapter>` guide.
    * You have installed either `Raspbian For Robots`_, the GoPiGo3 `repository`_ or the `GoPiGo3 package`_ (the pip package).
    * You have the ``gopigo3`` package installed by typing the command ``pip freeze | grep gopigo3`` on your Raspberry Pi's terminal. If the package is installed, then a string with the ``gopigo3==[x.y.z]`` format will show up.
 
 If you encounter issues that aren't covered by our :ref:`Getting Started <getting-started-chapter>` guide or :ref:`FAQ <faq-chapter>` chapter, please head over to our `forum`_.
 
+Take note that the easygopigo3 module depends on the underlying gopigo3 module.  The base gopigo3 module contains lower level calls that would allow you finer control over your GoPiGo3 and its functionality is available to you once easygopigo3 is imported.
+
```

Comment:
Can you use double tildas every time a module is mentioned?
Like instead of saying gopigo3, we use double tildas to put emphasis on the actual package. Again, not just one tilda on each side, but double the amount. 

Take a look on line 21 on how we did for the `gopigo3` package.

Reply:
Done